2021-12-29 15:04:39,628 - INFO - Database - end with duration 00:00 
2021-12-29 15:04:39,975 - INFO - view data - end with duration 00:00 
2021-12-29 15:04:39,995 - INFO - groupby - end with duration 00:00 
2021-12-29 15:04:40,327 - INFO - view data - end with duration 00:00 
2021-12-29 15:04:40,339 - CRITICAL - Html Jinja - Critical Error 
2021-12-29 15:04:40,339 - CRITICAL - Database - 'len' is undefined 
2021-12-29 15:04:40,340 - CRITICAL - Database - Traceback (most recent call last):
  File "/root/projects/bambou/nodes/__init__.py", line 167, in _run
    df = self.run(in1)
  File "/root/projects/bambou/nodes/htmljinja/main.py", line 22, in run
    html = withjinja(txt).render({"in1": in1})
  File "/usr/local/lib/python3.8/site-packages/jinja2/environment.py", line 1304, in render
    self.environment.handle_exception()
  File "/usr/local/lib/python3.8/site-packages/jinja2/environment.py", line 925, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "<template>", line 5, in top-level template code
  File "/usr/local/lib/python3.8/site-packages/jinja2/utils.py", line 84, in from_obj
    if hasattr(obj, "jinja_pass_arg"):
jinja2.exceptions.UndefinedError: 'len' is undefined
 
